apps/contrib/templates/a4_candy_contrib/includes/form_field.html

```diff
@@ -15,13 +15,15 @@
     {% endblock %}
     {% block field %}
         <div class="widget widget--{{ field|widget_type }}">
-          {% with describedby="hint_"|add:field.id_for_label %}
-          {% if field.errors %}
-                {% render_field field aria-invalid="true" aria-describedby=describedby %}
-          {% else %}
-            {% render_field field aria-describedby=describedby %}
+          {% if field %}
```

Comment:
The actual signup form is a bit hard to find, but you should change it there: https://github.com/liqd/adhocracy-plus/blob/8c444397ea6b3c2d8fb12f4f3ae8f9e808331477/adhocracy-plus/templates/account/signup.html#LL55C101-L55C101